tests/SendTelescopeReportTest.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+use Carbon\Carbon;
+use Illuminate\Support\Arr;
+use Illuminate\Support\Facades\Artisan;
+use RonasIT\TelescopeExtension\Tests\TestCase;
+use RonasIT\TelescopeExtension\TelescopeExtensionServiceProvider;
+use Illuminate\Support\Facades\Config;
+use Illuminate\Console\Scheduling\Schedule;
+use RonasIT\TelescopeExtension\Tests\Support\SQLMockTrait;
+use Illuminate\Support\Str;
+
+class SendTelescopeReportTest extends TestCase
+{
+    use SqlMockTrait;
+
+    protected TelescopeExtensionServiceProvider $serviceProvider;
+
+    protected Schedule $schedule;
+
+    public function setUp(): void
+    {
+        parent::setUp();
+
+        $this->serviceProvider = new TelescopeExtensionServiceProvider($this->app);
+
+        $this->schedule = $this->app->make(Schedule::class);
+
+        Config::set('app.name', 'Test app name');
+    }
+
+    public function testCommandInTheList()
+    {
+        $this->assertArrayHasKey('telescope:send-report', Artisan::all());
+    }
+
+    public function testCommandEnabledAndRun(): void
+    {
+        Config::set('telescope.notifications.report.enabled', true);
+        Config::set('telescope.notifications.report.frequency', 7);
+        Config::set('telescope.notifications.report.time', '15');
+
+        $this->serviceProvider->boot();
+
+        $event = Arr::first($this->schedule->events());
+
+        $this->assertTrue(Str::endsWith($event->command, 'telescope:send-report'));
+
+        $this->assertEquals('0 15 * * *', $event->getExpression());
```

Comment:
let's implement a separate helper for asserting it

```suggestion
        $this->assertScheduledEventEquals($event, 'telescope:send-report', '0 15 * * *');
```